docker-compose.yml

```diff
@@ -71,7 +71,7 @@ services:
 
     restart: always
     ports:
-      - 4000:4000
+      - 4001:4000
```

Comment:
ive seen the other dependency issues but this feels like it's because you had port 4000 occupied prior to running toolkit? can you just clear your ports before running this to see if that works?

Reply:
Yeah, I checked with lsof last Friday, but I had nothing in 4000. I'll double-check again 🙃 just in case

Reply:
I rebooted my laptop and rechecked the port and this time I found the nxd process (NoMachine) using port 4000 in my laptop (Don't know why didn't appeared first time). I'll revert the last commit.

```
sudo lsof -i TCP:4000
COMMAND PID USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME
nxd     595   nx    3u  IPv4 0x8312dde2fbb0b831      0t0  TCP *:terabase (LISTEN)
nxd     595   nx    4u  IPv6 0x8312dde7cb9787e1      0t0  TCP *:terabase (LISTEN)
```

Reply:
@francisco-perez-sorrosal thanks for making that change, It's approve but you'll have to sign your commits so we can merge the changes